docroot/themes/custom/uids_base/assets/js/uids_base.js

```diff
@@ -4,3 +4,19 @@
  */
 
 document.documentElement.className = document.documentElement.className.replace("no-js", "js");
+
+// @todo: Scope click event listener to only a.menuparents.
```

Comment:
Maybe this is ok? https://gomakethings.com/why-event-delegation-is-a-better-way-to-listen-for-events-in-vanilla-js/#web-performance